rolling/distribution.yaml

```diff
@@ -5625,7 +5626,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/cyberbotics/webots_ros2.git
-      version: master
+      version: rolling
```

Comment:
Stray edit? If not, the changes to webots_ros2 probably belong in a separate PR.